src/components/common/Alert/Alert.tsx

```diff
@@ -1,10 +1,23 @@
 import { useCallback, useState } from 'react'
+import type { ReactNode } from 'React'
 import UIAlert from 'src/components/ui/Alert'
 import { mark } from 'src/sdk/tests/mark'
-import type { PropsWithChildren } from 'react'
 import Icon from 'src/components/ui/Icon'
 
-function Alert({ children }: PropsWithChildren<unknown>) {
+interface Props {
+  icon: string
+  text: ReactNode
```

Comment:
Hmm, not so important but in the `ui/Alert` we render it with `data-alert-content`, what do you think about:
```suggestion
  content: ReactNode
```

Reply:
Like that. Going to discuss that with @filipeafns to change this in other CMS components too, so we can standardize.